design.md

```diff
@@ -50,14 +62,21 @@ The client must periodically call a keepalive function in the kernel. If the ker
 - When started headlessly, the kernel is configured to not have a timeout.
 - When started with a head, the kernel is configured to have a timeout.
 
+#### Display Operations
+
+- Set the display server address.
+  - The kernel will push data to the display server using its gRPC API.
```

Comment:
Alternate way of handling this: the display connects to the same gRPC server on the kernel as the IDE does. There is a separate API group for producing data for the display, all using gRPC streaming calls. This would enable more flexibility in design while (from what I can see) reducing complexity of implementation.

Reply:
As we discussed in Slack, you will make these changes.